build/python.prf

```diff
@@ -52,15 +60,15 @@ macx {
   # make sure that you have installed a matching python-dev package.
   
   PYTHON_CONFIG = $${PYTHON_DIR}/bin/python$${PYTHON_VERSION}-config
-  system($${PYTHON_CONFIG} --embed --libs) {
-    unix:LIBS += $$system($${PYTHON_CONFIG} --embed --libs)
-  } else: unix:LIBS += $$system($${PYTHON_CONFIG} --libs)  
-  unix:QMAKE_CXXFLAGS += $$system($${PYTHON_CONFIG} --includes)
+  PYTHON_CONFIG_OPTIONS_LIBS = --libs
+  equals(PYTHON_VERSION_MAJOR, 3): PYTHON_CONFIG_OPTIONS_LIBS += --embed
```

Comment:
Probably incompatible change for Python before 3.8. But Python 3.7 EOL is 27 Jun 2023.

Reply:
Python lifecycle https://devguide.python.org/versions/

Reply:
Ok, I'll rewrite this for PR as @he-hesce commented below https://github.com/MeVisLab/pythonqt/pull/89#issuecomment-1400176891